pyproject.toml

```diff
@@ -81,6 +81,8 @@ filterwarnings = [
   'ignore:setDaemon\(\) is deprecated, set the daemon attribute instead:DeprecationWarning',
   'ignore:TriCutTool is deprecated::',
   'ignore:Cut3dTool is deprecated::',
+  'ignore:This figure includes Axes that are not compatible with tight_layout:UserWarning',
+  'ignore:The figure layout has changed to tight:UserWarning',
```

Comment:
Why do these need to be ignored? It seems to me like Plopp should not raise warnings like this unless they are caused by the user.

Reply:
This is mostly to do with the tiled plots, where the axes handling is a little funny. Maybe it's better to catch the warnings in the code, so that we are sure this is the place they are coming from?

Reply:
> Maybe it's better to catch the warnings in the code, so that we are sure this is the place they are coming from?

:+1: